pretyping/reductionops.ml

```diff
@@ -536,12 +537,13 @@ let reduce_and_refold_cofix recfun env sigma cofix sk =
 let reduce_mind_case sigma mia =
   match EConstr.kind sigma mia.mconstr with
     | Construct ((ind_sp,i),u) ->
-(*	let ncargs = (fst mia.mci).(i-1) in*)
+(*      let ncargs = (fst mia.mci).(i-1) in*)
         let real_cargs = List.skipn mia.mci.ci_npar mia.mcargs in
         applist (mia.mlf.(i-1),real_cargs)
     | CoFix cofix ->
         let cofix_def = contract_cofix sigma cofix in
-        mkCase (mia.mci, mia.mP, applist(cofix_def,mia.mcargs), mia.mlf)
+        (* XXX Is NoInvert OK here? *)
+        mkCase (mia.mci, mia.mP, NoInvert, applist(cofix_def,mia.mcargs), mia.mlf)
```

Comment:
This case of a comatch with special reduction can probably happen, not sure if using NoInvert is a problem (maybe the result of this function gets immediately reduced? I don't understand this code)
Maybe we should just ban coinductives in sprop.